src/main/kotlin/org/rust/cargo/toolchain/impl/CargoMetadata.kt

```diff
@@ -42,7 +42,12 @@ object CargoMetadata {
         /**
          * Ids of packages that are members of the cargo workspace
          */
-        val workspace_members: List<String>?
+        val workspace_members: List<String>?,
+
+        /**
+         * Path to workspace root folder. Can be null for old cargo version
+         */
+        val workspace_root: String?
```

Comment:
Let's add `// BACKCOMPAT: Rust 1.23.0`, so that we remove `?` once we drop support for 1.23?